drivers/dma/dma_mcux_edma.c

```diff
@@ -651,10 +651,39 @@ static int dma_mcux_edma_init(const struct device *dev)
 			    irq_enable(DT_INST_IRQ_BY_IDX(n, idx, irq));	\
 	}
 
+#define DMA_CHANNLES_SHARED_IRQ_MASK(i, n) \
```

Comment:
Typo

Reply:
Can you be a little more specific?

Reply:
I think `CHANNLES` should be `CHANNELS`

Reply:
Yes, thank you, I will correct it later.

Reply:
Does the build give an error on this?

Reply:
No, build error is caused by dma_mcux_edma_error_irq_handler function defined but not used. However, IRQ_CONNECT function will include the function into interrupt function. I don't know how to resolve the building warning issue, and it seems a legacy issue. (There is standalone error IRQ on RT1180 edma.)